core/templates/dev/head/editor/ExplorationHistory.js

```diff
@@ -70,16 +112,25 @@ oppia.controller('ExplorationHistory', [
        */
       $scope.compareVersions = {};
       $scope.compareVersionMetadata = {};
+      
+      // Contains the IDs of the versions selected for comparison. Should contain a maximum of two elements.
+      $scope.selectedVersionsArray = []; 
 
       $scope.hideHistoryGraph = true;
-      $scope.hideCompareVersionsButton = false;
+
+      // Disable all comparisons if there are less than two revisions in total.
+      $scope.disableCompare = (currentVersion < 2);
+
+      $scope.hideCompareVersionsButton = $scope.disableCompare;
```

Comment:
The name reads like a function that does something, since it starts with a verb. Maybe call it $scope.compareVersionsButtonIsHidden?